mods/d2k/chrome/ingame-player.yaml

```
@@ -65,7 +65,7 @@ Container@PLAYER_WIDGETS:
 					Background: command-button
 					DisableKeySound: true
 					TooltipText: Force Move
-					TooltipDesc: Selected units will move to the desired location\n - Default activity for the target is suppressed\n - Vehicles will attempt to crush enemies at the target location\n\nLeft-click icon then right-click on target.\nHold {(Alt)} to activate temporarily while commanding units.
+					TooltipDesc: Selected units will move to the desired location\n - Default activity for the target is suppressed\n - Vehicles will attempt to crush enemies at the target location\n - Deployed units will undeploy and move to the target location\n\nLeft-click icon then right-click on target.\nHold {(Alt)} to activate temporarily while commanding units.
```

```suggestion
					TooltipDesc: Selected units will move to the desired location\n - Default activity for the target is suppressed\n - Vehicles will attempt to crush enemies at the target location\n - Deployed thumpers will undeploy and move to the target location\n\nLeft-click icon then right-click on target.\nHold {(Alt)} to activate temporarily while commanding units.
```
?
They are the only unit this works for. (Since you can't repack mcvs.)

---

I was unsure if we would rather consistency (i didn't actually check TD or RA, but this is the exact wording on TS i copy pasted) or specifying it. If you think this is better i can change.

---

I'd prefer explicitly using `thumpers`, so people don't get the impression it works on mcvs.